Sources/SwiftDocC/Model/Rendering/RenderContentCompiler.swift

```diff
@@ -130,7 +131,13 @@ struct RenderContentCompiler: MarkupVisitor {
         }
         
         collectedTopicReferences.append(resolved)
-        return [RenderInlineContent.reference(identifier: .init(resolved.absoluteString), isActive: true, overridingTitle: nil, overridingTitleInlineContent: nil)]
+        let linkTitleInlineContent = link.children.reduce(into: [], { result, child in result.append(contentsOf: visit(child))}) as! [RenderInlineContent]
```

Comment:
Is this different from `flatMap`?

```suggestion
        let linkTitleInlineContent = link.children.flatMap { visit($0) } as! [RenderInlineContent]
```

```

Reply:
I think flatMap is better. (Just copied from Line 91😂)

Reply:
```suggestion
        let linkTitleInlineContent =  link.children.compactMap { visit($0) as? [RenderInlineContent] }
```

Fixed